Gems/Archive/Code/Include/Archive/ArchiveInterfaceStructs.h

```diff
@@ -173,23 +244,54 @@ namespace Archive
         AZ::u64 m_offset : 39; // 64-bits
     };
 
+    static_assert(sizeof(ArchiveTocFileMetadata) == 16, "File Metadata size should be 16 bytes");
+
     //! Views an entry of a single File Path Index when the File Path Table of the Archive TOC
```

Comment:
this sentence reads a little odd - is this correct?
```suggestion
    //! Represents an entry of a single File Path Index within the File Path Table of the Archive TOC
```

Reply:
Yeah, your edit looks about right.

The ArchiveTocFilePathIndex is a structure used to store the size of a file path and an offset into the file path blob section of the Table of Contents.